KsApi/models/GraphCategory.swift

```diff
@@ -0,0 +1,132 @@
+import Foundation
+
+public struct ParentCategory: Swift.Decodable {
+  public let id: String
+  public let name: String
+
+  public var categoryType: RootCategoriesEnvelope.Category {
+      return RootCategoriesEnvelope.Category(
+        id: id,
+        name: name
+    )
+  }
+}
+
+public struct RootCategoriesEnvelope: Swift.Decodable {
+  public let rootCategories: [Category]
+
+  public struct Category: Swift.Decodable {
+    public let id: String
+    public var intID: Int? {
+        return decompose(id: id)
+    }
+
+    public static let gamesId: Int = 12
```

Comment:
Why this one hard-coded value?

Reply:
This code i copied from the old `Category` class. it's related to some logic to show or not a Rating Alert depending if the category is `Game`.